scripts/run-migrations-task.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+import argparse
+import time
+
+import boto3
+
+parser = argparse.ArgumentParser(description="Run ECS task with optional command.")
+parser.add_argument(
+    "--command",
+    type=str,
+    default="flask db upgrade",
+    help="Command to run in the ECS task (default: 'flask db upgrade').",
+)
+args = parser.parse_args()
+print(f"Starting script to run command: {args.command}")
+
+command = args.command.split()
```

Comment:
I'm guessing this won't work quite as we might expect with quoted strings (eg `./run-migrations-task --command "uv run python -c "import random; print(random.randint(1,100))"`), possibly just something to comment on/bear in mind. Can't think of a definitive example of a command we'd actually use with this problem, but you never know...

Also if this script can take arbitrary commands, `run-migrations-task.py` is probably a bad name? Maybe call it `ad-hoc-task` in line with the ecs cluster/task naming?

Reply:
> if this script can take arbitrary commands, run-migrations-task.py is probably a bad name? Maybe call it ad-hoc-task in line with the ecs cluster/task naming?

@samuelhwilliams fair point! I've renamed in: 71f8eef